packages/types/lib/terra/index.ts

```diff
@@ -0,0 +1,21 @@
+import { SendOptions } from '../chain'
+
+export interface TerraSendOptions extends SendOptions {
+  messages?: any[]
+}
+
+export interface InputTransaction {
+  buyer?: string
+  seller?: string
+  expiration?: number
+  value?: number
+  secret_hash?: string
+  secret?: string
+  contractAddress?: string
+  method?: {
+    claim?: {
+      secret: string
+    }
+    refund: () => void
```

Comment:
is this supposed to be a function?

Reply:
Yes, so by default claim and refund both are functions which are passed to the contract, I've used function for type because `refund: object` or `refund: {}` is not liked by the linter and in js/ts function are objects as well, that why I'm using function, even more I think should convert claim to function as well :)